### ISSUE TITLE
False positive for `CKV_SECRET_6` for `google_secret_manager_secret_version` resources

### ISSUE THREAD

**Issue by calexandre** [bridgecrewio/checkov#1660]
**Describe the bug**
Checkov 2.0.446 is detecting a false positive regarding the secret manager usage.

Basically it is showing multiple failed check for resources declared in the following format:

```hcl
data "google_secret_manager_secret_version" "mysecret" {
  secret = "somesecretid"
}
```

With the error:

```
Check: CKV_SECRET_6: "Base64 High Entropy String"
        FAILED for resource: 8b50635275125444f17b28140b0eb5f529861e4b
        File: /secrets.tf:6-7
        Guide: https://docs.bridgecrew.io/docs/git_secrets_6
```

**Expected behavior**
Checkov should not detect this is a failed check, because this is the secret-id, and not the secret value.

**Desktop (please complete the following information):**
 - OS: ubuntu
 - Checkov Version  2.0.446

**Comment by ddanf** [comment]
I was getting this in jenkins but when I ran on desktop without the -o and redirect, I discovered that I was really getting a ton of file parse errors (https://github.com/bridgecrewio/checkov/issues/1663).  limiting to 2.0.437 has resolved the issue for now.

**Comment by bogdan-milescu** [comment]
I can confirm that this is still present. I'm using version: 2.0.563.

**Comment by SebastianBalle** [comment]
I am getting the same error when reading the following ARM template which is located in my directory.

```json
{
    "properties": {
        "type": "AzureBlobStorage",
        "annotations": [],
        "typeProperties": {
            "connectionString": {
                "type": "AzureKeyVaultSecret",
                "store": {
                    "referenceName": "kv_ls_primary",
                    "type": "LinkedServiceReference"
                },
                "secretName": "storage-account-connection-string"
            }
        }
    },
    "type": "Microsoft.DataFactory/factories/linkedservices"
}
```
The following error was received
Resource 2b442dd9a7ae593d71f647e89bc3f809e01510bb failed in check CKV_SECRET_6/Base64 High Entropy String - /initialization/03-AzureStorageProcessingLinkedService.json:[12, 13] - Guideline: https://docs.bridgecrew.io/docs/git_secrets_6

I am using version: 2.0.616

**Comment by codyhazelwood** [comment]
I get this with the github_actions_secret resource's `secret_name` as well.

**Comment by nimrodkor** [comment]
Thanks for all the examples guys. I promise we are planning to take a look at this in the few upcoming weeks!
